expat/lib/xmlparse.c

```diff
@@ -133,7 +133,7 @@
       * BSD / macOS (including <10.7) (arc4random): HAVE_ARC4RANDOM, \
       * libbsd (arc4random_buf): HAVE_ARC4RANDOM_BUF + HAVE_LIBBSD, \
       * libbsd (arc4random): HAVE_ARC4RANDOM + HAVE_LIBBSD, \
-      * Linux (including <3.17) / BSD / macOS (including <10.7) (/dev/urandom): XML_DEV_URANDOM, \
+      * Linux (including >=1.3) / BSD / macOS (including <10.7) / Solaris >=8 (/dev/urandom): XML_DEV_URANDOM, \
```

Comment:
@noloader I saw your comment at https://github.com/libexpat/libexpat/pull/573#discussion_r815348431. Please revert the `including >=1.3` back to `including <3.17`. The point/focus here is that in contrast to things mentioned further up, this will not require `Linux >=3.17` and hence is an option in those cases. Given how long ago Linux 1.3 is and that we require a C99 compiler and all, I don't see much non-archival practical value in `>=1.3` here either. Let's go back to `<3.17` please. Thank you!